python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -15,7 +15,7 @@
 logger = getLogger()
 
 _reg_id_str = '_registry_id'
-_id_str = 'id'
+_id_str = '_id'
```

Comment:
We have always stored the registry_id within the object. If this changes now it potentially represents a __HUGE__ rewrite in how the registry works imo.

If we want to, then great, but I'm just trying to reduce the total amount of code and the ```GangaObject._id``` and ```GangaObject._registry_id``` were both sources of obvious duplication of information.

I'm strongly in favor of keeping the registry id and the ```.id``` from within the Schema separate as it will just avoid headaches. This is also the way the registry has been designed to work for a __long__ time now.

The ```_id_str``` was used in the past and I'm now wanting to remove this as it was defined in the ```GangaRepository.py```. It likely was a historical mistake that the object ```.id``` became confused with the object ```._id``` I feel from looking at the code over time.

I'm working on reducing the information in the registry here as doing some 'sane _little_ changes' within GangaObject are causing lockups (which I still cannot produce elsewhere) in the Jenkins tests.